picky-asn1/src/wrapper.rs

```diff
@@ -200,6 +200,7 @@ asn1_wrapper! { auto struct IA5StringAsn1(IA5String),               Tag::IA5_STR
 asn1_wrapper! { auto struct BMPStringAsn1(BMPString),               Tag::BMP_STRING }
 asn1_wrapper! { auto struct UTCTimeAsn1(UTCTime),                   Tag::UTC_TIME }
 asn1_wrapper! { auto struct GeneralizedTimeAsn1(GeneralizedTime),   Tag::GENERALIZED_TIME }
+asn1_wrapper! { auto struct GeneralStringAsn1(IA5String),           Tag::GENERAL_STRING }
```

Comment:
I see you just wrapped the IA5String which is surprising to me. Is that how the GeneralString is defined?

Reply:
I just took the `GeneralString` definition from the `Kerberos` ASN.1 grammar specification(It's said that `GeneralString` contains `IA5String`).

Reply:
OK, I guess we support a _subset_ then. Can you add some comment about that?